### ISSUE TITLE
JavadocMethod false positive when @{code ...} is at the beginning of the line

### ISSUE THREAD

**Issue by pbaranchikov** [checkstyle/checkstyle#2267]
Checkstyle reports "First sentence should end with a period" for the following Javadoc block:
```java
        /**
         * Method do something on, when parameter is
         * @{code null}.
         */
```
The following is treated without false positive:
```java
        /**
         * Method do something on, when parameter is @{code null}.
         */
```
